src/NHSD.BuyingCatalogue.Identity.Api/Startup.cs

```diff
@@ -131,6 +136,15 @@ public void ConfigureApp(IApplicationBuilder app)
             {
                 endpoints.MapDefaultControllerRoute();
                 endpoints.MapControllers();
+                endpoints.MapHealthChecks("/health/live", new HealthCheckOptions
+                {
+                    Predicate = (healthCheckRegistration) => healthCheckRegistration.Tags.Contains(HealthCheckTags.Live)
```

Comment:
The brackets around the lambda parameter can be omitted here.

Reply:
🤦‍♂ D'oh! I'll get rid of them.